sdcm/cluster_k8s/eks.py

```diff
@@ -703,6 +703,13 @@ def ec2_host(self):
     def ec2_instance_id(self):
         return self._node.spec.provider_id.split('/')[-1]
 
+    @property
```

Comment:
It should be defined in the `sdcm/cluster_k8s/__init__.py` module.
Only there, so it could be inherited.

Reply:
moved